transport/http/handler_test.go

```diff
@@ -79,6 +87,85 @@ func TestHandlerSucces(t *testing.T) {
 	assert.Equal(t, rw.Body.String(), "")
 }
 
+func TestHandlerHeaders(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	tests := []struct {
+		giveHeaders map[string]string // http headers to send
+
+		wantTTL     time.Duration
+		wantHeaders transport.Headers
+		wantBaggage transport.Headers
+	}{
+		{
+			giveHeaders: map[string]string{
```

Comment:
This is trying to differentiate between the YARPC headers and the HTTP headers. The `map[string]string` is the HTTP headers. The `transport.Headers` is the YARPC headers.